.circleci/config.base.yml

```diff
@@ -542,13 +519,40 @@ jobs:
       - restore_cache:
           keys:
             - amplify-cli-ssh-deps-{{ .Branch }}
+      - run:
+          name: Upload Pkg Binary
+          command: |
+            yarn yarn-use-bash
+            source .circleci/local_publish_helpers.sh
+            setNpmTag
+            yarn update-versions --preid=$NPM_TAG --conventional-prerelease
+            startLocalRegistry "$(pwd)/.circleci/verdaccio.yaml"
+            setNpmRegistryUrlToLocal
+            git config user.email not@used.com
+            git config user.name "Doesnt Matter"
+            changeNpmGlobalPath
+            yarn yarn-use-bash
+            yarn pkg-all
+            unsetNpmRegistryUrl
+            git stash
+            uploadPkgCli
+            ./out/amplify-pkg-linux --version
       - run:
           name: Authenticate with npm
           command: echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > ~/.npmrc
       - run:
           name: Publish Amplify CLI
           command: |
             bash ./.circleci/publish.sh
+      - run:
```

Comment:
this is a new sanity check that ensures the npm package was deployed properly and it's postinstall hook is able to download the appropriate binary.